src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java

```diff
@@ -750,6 +756,236 @@ public static void buildNotesTableView(TableView<UserNoteFX> tableView, Observab
         tableView.getColumns().add(updateTimeColumn);
     }
 
+    public static void buildTutorialTable(TableView<TutorialFx> tutorialTableView, TutorialService tutorialService, Logger log, Runnable refresh) {
```

Comment:
Please add copy & paste behavior as is done now in all controls in develop. See examples there.